features/org.wso2.carbon.permissions.rest.api.feature/pom.xml

```diff
@@ -0,0 +1,230 @@
+                                        <include>resources/**</include>
+                                        <include>tmp/**</include>
+                                        <include>p2.inf</include>
+                                        <include>build.properties</include>
+                                    </includes>
+                                </resource>
+                            </resources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <plugin>
+                <groupId>org.wso2.carbon.maven</groupId>
+                <artifactId>carbon-feature-plugin</artifactId>
+                <version>${carbon.feature.plugin.version}</version>
+                <extensions>true</extensions>
+                <executions>
+                    <execution>
+                        <id>1-p2-feature-generation</id>
+                        <goals>
+                            <goal>generate</goal>
+                        </goals>
+                        <configuration>
+                            <propertyFile>../etc/feature.properties</propertyFile>
+                            <adviceFileContents>
+                                <advice>
+                                    <name>org.wso2.carbon.p2.category.type</name>
+                                    <value>server</value>
+                                </advice>
+                                <advice>
+                                    <name>org.eclipse.equinox.p2.type.group</name>
+                                    <value>true</value>
+                                </advice>
+                            </adviceFileContents>
+                            <bundles>
+                                <bundle>
+                                    <symbolicName>org.wso2.carbon.permissions.rest.api</symbolicName>
+                                    <version>${carbon.analytics.version}</version>
+                                </bundle>
+                                <bundle>
+                                    <symbolicName>org.wso2.carbon.analytics.permissions</symbolicName>
+                                    <version>${carbon.analytics-common.version}</version>
+                                </bundle>
+                                <bundle>
+                                    <symbolicName>antlr4-runtime</symbolicName>
+                                    <version>${antlr4.runtime.version}</version>
+                                </bundle>
+                                <bundle>
+                                    <symbolicName>com.google.gson</symbolicName>
+                                    <version>${gson.version}</version>
+                                </bundle>
+                                <bundle>
+                                    <symbolicName>org.wso2.carbon.stream.processor.core</symbolicName>
```

Comment:
I think we do not need bundle all these dependencies in the permission feature. Shall we remove the unnecessary dependencies from it?

Reply:
Fixed in [#f464715d8a1cc46354269aef081db46e40248555](https://github.com/wso2/carbon-analytics/commit/f464715d8a1cc46354269aef081db46e40248555)